docs/integrations/index.rst

```diff
@@ -40,6 +40,7 @@ To install a plugin just include the plugin **after** Raven has been loaded and
    ember
    react
    vue
+   raven-testkit
```

Comment:
Are you aware that it'll only show up here https://docs.sentry.io/clients/javascript/integrations/ and not in the sidebar? I'd not add it to the sidebar anyway, as it's more a helpful tool than framework integration.
Let's maybe add a section in in `Tips and Tricks` that can mention it and link to the integration page?

Reply:
Or just completely move it to `Tips and Tricks` section instead of a separate page? As, as I've mentioned before, it's not really an integration :)

Reply:
ok. will move it there :)